llvm/test/CodeGen/Thumb2/pacbti-m-indirect-tail-call-no-r12.ll

```diff
@@ -0,0 +1,78 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
```

Comment:
If you're going to hand-edit the CHECK lines, please mark the file with "NOTE: Do not autogenerate" (see https://llvm.org/docs/TestingGuide.html#generating-assertions-in-regression-tests) .

In this case, I'm not sure it's worth hand-editing the CHECK lines.